### PR TITLE
Adding reference lines to plot panel docs

### PR DESCRIPTION
**Pull request by banisadr**
### Changelog
None

### Description

Adds documentation for reference lines in the plot panel. Resolves FG-6262.

### CODE REVIEW

File: static/img/docs/visualization/panels/plot/reference-plot.png
Comment (jtbandes) [inline static/img/docs/visualization/panels/plot/reference-plot.png:1]: would recommend trying `pngcrush` or `cwebp` (`brew install webp`) to make the image file smaller. Seems like we are trending towards webp for most of our images.

**Review by jtbandes — APPROVED** [review]

File: static/img/docs/visualization/panels/plot/reference-plot.png
Comment (banisadr) [reply static/img/docs/visualization/panels/plot/reference-plot.png:1]: Lovely, thank you

Comment (jtbandes) [inline static/img/docs/visualization/panels/plot/reference-plot.png:1]: nice but you forgot to commit the webp file 😅

Comment (jtbandes) [reply static/img/docs/visualization/panels/plot/reference-plot.png:1]: happily, CI caught this 🙌

**Pull request merged**